src/Main/registerTclCommands.cpp

```diff
@@ -213,6 +215,7 @@ void registerBasicBatchCommands(FOEDAG::Session* session) {
   session->TclInterp()->registerCmd("help", help, 0, 0);
 }
 
+#include <QDebug>
```

Comment:
this should be in the include section

Reply:
I intend on removing that and the DemoSettings command from the file once the interface is finished up.